GOVERNANCE.md

```diff
@@ -0,0 +1,28 @@
+# auth-gateway Governance
+
+This document defines project governance for the project.
+
+## Voting
+
+todo
```

Comment:
Since currently there are only two maintainers, I am not sure how to fill this section